webui/src/app/status/version/version.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+import * as generated from "../../graphql/generated";
+import { GraphQLService } from 'src/app/graphql/graphql.service';
+
+const defaultVersionName = 'N/A';
```

Comment:
I think maybe default to an empty string as you might get a flash of `N/A` on initial load with this?

Reply:
Sure, I will fix this later.